policy/slemicro/k3s.fc

```diff
@@ -5,6 +5,7 @@
 /usr/local/lib/systemd/system/k3s.*                             --  gen_context(system_u:object_r:container_unit_file_t,s0)
 /usr/s?bin/k3s                                                  --  gen_context(system_u:object_r:container_runtime_exec_t,s0)
 /usr/local/s?bin/k3s                                            --  gen_context(system_u:object_r:container_runtime_exec_t,s0)
+/opt/k3s/bin/k3s                                                --  gen_context(system_u:object_r:container_runtime_exec_t,s0)
```

Comment:
I think the difference here is that k3s actually installs the binary in /opt/bin/k3s not in /opt/k3s/bin/k3s 

https://github.com/k3s-io/k3s/blob/master/install.sh#L227-L234